composer.json

```diff
@@ -27,43 +27,43 @@
   ],
   "require": {
     "composer/installers": "^1.11",
-    "mautic/core-composer-scaffold": "4.x-dev",
-    "mautic/core-project-message": "4.x-dev",
-    "mautic/core-lib": "4.4.x-dev",
-    "mautic/grapes-js-builder-bundle": "4.4.x-dev",
-    "mautic/plugin-citrix": "4.4.x-dev",
-    "mautic/plugin-clearbit": "4.4.x-dev",
-    "mautic/plugin-cloudstorage": "4.4.x-dev",
-    "mautic/plugin-crm": "4.4.x-dev",
-    "mautic/plugin-emailmarketing": "4.4.x-dev",
-    "mautic/plugin-focus": "4.4.x-dev",
-    "mautic/plugin-fullcontact": "4.4.x-dev",
-    "mautic/plugin-gmail": "4.4.x-dev",
-    "mautic/plugin-outlook": "4.4.x-dev",
-    "mautic/plugin-social": "4.4.x-dev",
-    "mautic/plugin-tagmanager": "4.4.x-dev",
-    "mautic/plugin-zapier": "4.4.x-dev",
-    "mautic/theme-aurora": "4.4.x-dev",
-    "mautic/theme-blank": "4.4.x-dev",
-    "mautic/theme-brienz": "4.4.x-dev",
-    "mautic/theme-cards": "4.4.x-dev",
-    "mautic/theme-coffee": "4.4.x-dev",
-    "mautic/theme-confirmme": "4.4.x-dev",
-    "mautic/theme-fresh-center": "4.4.x-dev",
-    "mautic/theme-fresh-fixed": "4.4.x-dev",
-    "mautic/theme-fresh-left": "4.4.x-dev",
-    "mautic/theme-fresh-wide": "4.4.x-dev",
-    "mautic/theme-goldstar": "4.4.x-dev",
-    "mautic/theme-mauve": "4.4.x-dev",
-    "mautic/theme-nature": "4.4.x-dev",
-    "mautic/theme-neopolitan": "4.4.x-dev",
-    "mautic/theme-oxygen": "4.4.x-dev",
-    "mautic/theme-paprika": "4.4.x-dev",
-    "mautic/theme-skyline": "4.4.x-dev",
-    "mautic/theme-sparse": "4.4.x-dev",
-    "mautic/theme-sunday": "4.4.x-dev",
-    "mautic/theme-vibrant": "4.4.x-dev",
-    "mautic/theme-trulypersonal": "4.4.x-dev"
+    "mautic/core-composer-scaffold": "4.4.12",
+    "mautic/core-project-message": "4.4.12",
+    "mautic/core-lib": "4.4.4.12",
```

Comment:
Shouldn't it be `4.4.12` instead of `4.4.4.12`?

Reply:
Did I do it again? Funny ho a find-replace go wrong. Or sad at this point...

Reply:
Fixed in the second commit.